Makefile

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+.PHONY: xrefcheck test lint clean bats all
+
+# Build target from the common utility Makefile
+MAKEU = $(MAKE) -C make/
+
+MAKE_PACKAGE = $(MAKEU) PACKAGE=xrefcheck
+
+xrefcheck:
+	$(MAKE_PACKAGE) dev
+
+# Runs all tests.
+test:
+	make test-unit
+	make test-links
+	make bats
+
+test-dumb-term:
+	$(MAKE_PACKAGE) test-dumb-term
+
+test-hide-successes:
+	$(MAKE_PACKAGE) test-hide-successes
+
+clean:
+	$(MAKE_PACKAGE) clean
+
+lint:
+	hlint .
+
+####################################
+# Individual test suites
+
+test-unit:
+	$(MAKEU) test PACKAGE="xrefcheck:test:xrefcheck-tests"
+
+test-links:
```

Comment:
Does this target work correctly for you? When I run it, the tests fail - there seems to be something wrong with the FTP server...
![image](https://user-images.githubusercontent.com/718166/190864815-339cb251-3052-415b-9a24-d8bdde363999.png)

Reply:
Weird, everything works for me. :thinking: 

![image](https://user-images.githubusercontent.com/112060595/190876115-3e87ad19-03b1-4e4d-abab-00bf7b270a84.png)

Reply:
Actually, I took these settings from CI
https://github.com/serokell/xrefcheck/blob/f4eb187f31918367f71b741a3066f0632070d394/xrefcheck.nix#L23-L35

Reply:
Uh weird :/ I even tried it in a nix shell like in your screenshot :/ Must be something related to my setup... Nevermind then ^^